changelog/8.8.0.yaml

```diff
@@ -0,0 +1,320 @@
+version: 8.8.0
+entries:
+    - kind: enhancement
+      summary: async download of elastic-agent upgrade artifacts during fleet managed upgrade
+      description: ""
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2205
+      issue: []
+      timestamp: 1674840786
+      file:
+        name: 1674840786-async-download-upgrade-artifacts.yaml
+        checksum: 0aa7b1bb3810df922c917dc5015b16a5a4a6d965
+    - kind: feature
+      summary: Adds new StateWatch GRPC for watching the current state of the running Elastic Agent
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2252
+      issue: []
+      timestamp: 1676571093
+      file:
+        name: 1676571093-control-state-watch.yaml
+        checksum: 6f7a04fdd535f0c6de37545408e1d4dc4ffde6f7
```

Comment:
According to me, this change shouldn't be communicated to our users: it's not affecting them and will be use for our own purpose.